Header      hdr
int32[3]    del_pos
